From 7c1ad4544e5b7c48193623e2533506bf5655cb7d Mon Sep 17 00:00:00 2001
From: "Gururaj, Patil" <gururajx.patil@intel.com>
Date: Wed, 23 Sep 2020 13:59:06 +0530
Subject: [PATCH] Fix for CtsDevicePolicyManagerTestCases

testAdminActionBookkeeping

Actviitymanager is starting the service bugreport(/system/bin/dumpstate)
but we dont have this Service in init.rc file.
added the service in init.rc file by making bugreport true in mixin spec

Change-Id: I8cf4b8348c9a299740060069c0f19ba66413fa7f
Tracked-On: OAM-91820
Signed-off-by: Gururaj, Patil <gururajx.patil@intel.com>

diff --git a/caas/mixins.spec b/caas/mixins.spec
index 68b063b..8a472e2 100755
--- a/caas/mixins.spec
+++ b/caas/mixins.spec
@@ -87,3 +87,4 @@ firmware: true(all_firmwares=true)
 aaf: true
 suspend: auto
 sensors: mediation(disable_static_sensor_list=true)
+bugreport: true
--
2.17.1

